website/views/pages/query-detail.ejs

```diff
@@ -1,11 +1,20 @@
-<div id="query-detail" v-cloak>
-
-  <h1>TODO: Implement this page.</h1>
-  <p>(See also <em>assets/styles/pages/query-detail.less</em>, <em>assets/js/pages/query-detail.page.js</em>, and <em>api/controllers/view-query-detail.js</em>.)</p>
-
-  <p>This paragraph, and the content above it, are just here for reference.  The HTML generated from markdown is below.</p>
-
-  <%- partial('../partials/built-from-markdown/handbook-queries/get-installed-macos-software.ejs') %>
-
+<div class="d-flex justify-content-center">
+  <div class="col-6 my-5">
+      <div id="query-detail" v-cloak>
+        <%- partial('../partials/built-from-markdown/handbook-queries/get-installed-macos-software.ejs') %>
+      </div>
+  </div>
+  <div class="col-2 mx-5 my-5"><!-- __LANG=%%__ -->
+    <h4 id="support" permalink="support">Platforms</h4>
+    <p><i class="fa fa-apple fa-lg mr-1" alt="Mac"></i>
+      <i class="fa fa-windows fa-lg mr-1" alt="Windows"></i>
+      <i class="fa fa-linux fa-lg mr" alt="Linux"></i>
+    </p>
+    <h4 id="purpose"-1 permalink="purpose">Purpose</h4>
```

Comment:
Is the closing quote misplaced for the id prop? I think that may be causing the build test failure.  See proposed fix below.

```suggestion
    <h4 id="purpose-1" permalink="purpose">Purpose</h4>
```